src/VisualStudio/Core/Def/Implementation/Venus/ContainedDocument.cs

```diff
@@ -213,11 +223,11 @@ public string Name
             {
                 try
                 {
-                    return Path.GetFileName(this.FilePath);
+                    return Path.GetFileName(Key.Moniker);
```

Comment:
Previously the Key.Moniker was being used interchangeably with FilePath.  For TypeScript host types we're currently appending a .js extension to the file path so that the contained documents will be processed as JavaScript instead of TypeScript.  We now want all file path references to use FilePath but the display name to use the original moniker.